pkg/kubelet/cm/cpumanager/cpu_assignment.go

```diff
@@ -42,44 +42,44 @@ func newCPUAccumulator(topo *topology.CPUTopology, availableCPUs cpuset.CPUSet,
 	}
 }
 
-func (a *cpuAccumulator) take(cpus cpuset.CPUSet) {
-	a.result = a.result.Union(cpus)
-	a.details = a.details.KeepOnly(a.details.CPUs().Difference(a.result))
-	a.numCPUsNeeded -= cpus.Size()
+func (accumulator *cpuAccumulator) take(cpus cpuset.CPUSet) {
```

Comment:
`func (a *cpuAccumulator)` is the typical Golang style, I don't think it's meaningful to make this change ...

Also, any specific reason you want to use `int32` instead of `int`?

Reply:
since cpu cores and sockets must  be larger then 0, I just follow common practise. so using uint32 is much better.